backend/src/openbeheer/health_checks/checks.py

```diff
@@ -0,0 +1,140 @@
+import traceback
+from abc import ABC, abstractmethod
+
+from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+
+from requests.exceptions import ConnectionError, HTTPError
+from zgw_consumers.client import build_client
+from zgw_consumers.constants import APITypes
+from zgw_consumers.models import Service
+
+from .types import HealthCheckError, HealthCheckResult
+
+
+class HealthCheck(ABC):
+    def get_name(self):
+        return self.__class__.__name__
+
+    @abstractmethod
+    def run(self) -> HealthCheckResult: ...
+
+
+class ServiceHealthCheck(HealthCheck):
+    def get_name(self) -> str:
+        return _("Services")
+
+    def run(self) -> HealthCheckResult:
+        errors = []
+
+        for needed_service_type in settings.ZGW_REQUIRED_SERVICE_TYPES:
+            service = Service.objects.filter(api_type=needed_service_type).first()
+
+            if not service:
+                errors.append(
+                    HealthCheckError(
+                        code="missing_service",
+                        message=_('Missing service of type "{service_type}".').format(
+                            service_type=needed_service_type
+                        ),
+                        severity="error",
+                    )
+                )
+                continue
+
+        for service in Service.objects.all():
+            if service.connection_check != 200:
```

Comment:
Shouldn't we check for < 400?

From the [documentation for requests](https://docs.python-requests.org/en/latest/api/#requests.Response.ok)

> property ok
Returns True if [status_code](https://docs.python-requests.org/en/latest/api/#requests.Response.status_code) is less than 400, False if not.